operations/mimir-rules-action/entrypoint.sh

```diff
@@ -73,8 +73,8 @@ case "${ACTION}" in
 esac
 
 SINGLE_LINE_OUTPUT=$(echo "${OUTPUT}" | awk 'BEGIN { RS="%0A" } { gsub(/%/, "%25"); gsub(/\r/, "%0D"); gsub(/\n/, "%0A") } { print }')
-echo ::set-output name=detailed::"${SINGLE_LINE_OUTPUT}"
+echo "detailed=${SINGLE_LINE_OUTPUT}" >> $GITHUB_OUTPUT
 SUMMARY=$(echo "${OUTPUT}" | grep Summary)
-echo ::set-output name=summary::"${SUMMARY}"
+echo "summary=${SUMMARY}" >> $GITHUB_OUTPUT
```

Comment:
I think we need to do something different here to deal with multi-line `SUMMARY` values.

https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#multiline-strings suggests the following syntax:

```
{name}<<{delimiter}
{value}
{delimiter}
```

where `{delimiter}` is a randomly-generated value for each run

Reply:
That’s in the Github_env section but not in the output one. I’ll try out printing the output in one of our runs

Reply:
You were right (there's an issue about it not being documented: https://github.com/github/docs/issues/21529), I implemented it

I used a static delimiter (`EOF`) because using a random delimiter seems to be for protection against injection, which I don't think is a risk here

Reply:
Whats a delimiter that could definitely not be part of the summary? I think that if there's an EOF in the summary (probably has to be on its own line), it would mess up the output

Reply:
> Whats a delimiter that could definitely not be part of the summary?

Something like `END-OF-SUMMARY` perhaps?